lib/Raven/Serializer.php

```diff
@@ -67,17 +67,44 @@ public function __construct($mb_detect_order = null)
      * sanitization and encoding.
      *
      * @param mixed $value
+     * @param array $context
+     * @return string|bool|double|int|null|object|array
```

Comment:
> Doc is still missing

I don\'t know what should I write. And I am afraid that my line should be corrected because english is not my native language. I know it isn\'t yours either.